Submissions/St-shaddai.json

```diff
@@ -0,0 +1,16 @@
+{
+    "name": "Emmanuel Olatunbosun",
+    "img": "",
+    "email": "tolaniemmanuel12@gmail.com",
+    "links": {
+      "website": "[YOUR_WEBSITE_URL]",
```

Comment:
Please remove the placeholder content, see instructions in `README.md`.

Reply:
Placeholder has been removed